app/src/test/java/org/simple/clinic/home/patients/illustration/MonthRoomTypeConverterTest.kt

```diff
@@ -0,0 +1,20 @@
+package org.simple.clinic.home.patients.illustration
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.threeten.bp.Month
+
+class MonthRoomTypeConverterTest {
```

Comment:
I meant adding a test for the `DayOfMonth.MoshiTypeAdapter`. Something that exercises the entire (de)serialisation of the "01 January" string. Could you please update this test to handle this?

Reply:
done